app/presenters/local_authority_presenter.rb

```diff
@@ -8,4 +8,19 @@ def homepage_status
   def homepage_link_last_checked
     homepage_url.blank? ? "" : last_checked
   end
+
+  def display_status
+    return "inactive" unless active?
+    return "active, but being retired" if active_end_date
+
+    "active"
+  end
+
+  def display_end_notes?
+    !(active? && active_end_date.nil?)
+  end
+
+  def display_end_title
```

Comment:
status will break it, because of a dubious existing design choice for the LocalAuthority model that means we already have a status field we can't rename. But I've changed these a bit to be more in keeping with the names of the fields and hopefully more useful:

- display_status -> authority_status
- display_end_notes? -> should_display_end_notes?
- display_end_title -> active_end_date_title

Reply:
Looks good